packages/frontend/src/view/pages/user/UserPage.tsx

```diff
@@ -55,7 +55,7 @@ function UserPage(props: UserPageProps) {
   return (
     <Page
       path={`/users/${props.starkKey.toString()}`}
-      description="TODO: description"
+      description={`User page of ${props.starkKey.toString()}`}
```

Comment:
A longer description for user page could be good for SEO. My suggestion here:

"Details of user {starkKey} including assets, history and transactions"